Client/metrics.yaml

```diff
@@ -3419,3 +3419,18 @@ places_history_migration:
       - teshaq@mozilla.com
       - sync-team@mozilla.com
     expires: "2023-10-17"
+
+# Credit card autofill
+credit_card:
+  autofill_settings_tapped:
+    type: event
+    description: |
+      Recorded when the user taps on credit card autofill
+      settings item in settings screen.
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-5394
+    data_reviews:
+      - https://mozilla-hub.atlassian.net/browse/FXIOS-5394
```

Comment:
Shouldn't the data review link be this PR? So `https://github.com/mozilla-mobile/firefox-ios/pull/12813`?

Reply:
Can we also set the bug to the PR because it is the public one, as opposed to pointing to a link most people can't access.

Reply:
@dnarcese its a story so not public afaik, but let me know if you have a public link and I will update

Reply:
Sorry I meant use the PR link again

Reply:
Oh yes of course, just pushed that change :)